src/pal/src/debug/debug.cpp

```diff
@@ -530,7 +530,27 @@ RtlRestoreContext(
   IN PEXCEPTION_RECORD ExceptionRecord
 )
 {
-    ASSERT("UNIXTODO: Implement this");
+    native_context_t ucontext;
+
```

Comment:
This won't work on OSX.  We don't have native_context_t, we probably want CONTEXT_SetThreadContextOnPort here.